src/sass/layout/global.scss

```diff
@@ -29,4 +32,5 @@ a,
 p {
     font-family: $font-sans-serif;
     font-weight: 400;
+    text-decoration: none;
```

Comment:
element p posiada domyslnie text-decoration: none i nie musisz tego jeszcze raz pisac ;) -> https://www.w3schools.com/cssref/pr_text_text-decoration.asp

Reply:
nad elementem p jest jeszcze element a

Reply:
ahhh, pominalem go :)

W takim razie to powinno wygladac tak:

a,
p {
    font-family: $font-sans-serif;
    font-weight: 400;
}
a {
 text-decoration: none;
}

Reply:
jest OK :)